src/web/app/lib/fetchPatientData.ts

```diff
@@ -93,7 +93,7 @@ export async function fetchParticipantId(
   const correlationId = crypto.randomUUID();
 
   try {
-    const url = `${process.env.EXPERIENCE_API_URL}/api/participant`;
+    const url = `${process.env.EXPERIENCE_API_URL}/api/participant/me/id`;
```

Comment:
Does this need to be `participants`? with an s

Reply:
changed